framework/decode/vulkan_decoder_base.cpp

```diff
@@ -515,5 +536,10 @@ void VulkanDecoderBase::DispatchSetTlasToBlasDependencyCommand(format::HandleId
     }
 }
 
+void VulkanDecoderBase::SetCurrentBlockIndex(uint64_t block_index)
+{
+    block_index_ = block_index;
```

Comment:
In order to have consistency between the dx12 and the Vulkan base consume classes this should be implemented the same way it is implemented in the `Dx12DecoderBase`:
```
void Dx12DecoderBase::SetCurrentBlockIndex(uint64_t block_index)
{
    for (auto consumer : consumers_)
    {
        consumer->SetCurrentBlockIndex(block_index);
    }
}
```